### PR TITLE
Update to Tabris.js 3.4

### PR DESCRIPTION
**Pull request by cpetrov**
Move example to .example to enable builds that were previously not
possible due to a Cordova limitation [1]. Note that the example build
only works with Tabris CLI <= 3.2 since it does not work with the newer
Cordova versions supplied with Tabris CLI 3.3+. Long-term solution
pending.

[1]: apache/cordova-cli#401

Change-Id: I77a4fe00b526f5aaa164a5e52d1cecb51adcd72b

### CODE REVIEW

File: src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import com.eclipsesource.tabris.android.ObjectHandler
+import com.eclipsesource.v8.V8Object
+
+class GCMHeartbeatHandler(private val scope: ActivityScope)
+    : ObjectHandler<GCMHeartbeat> {
+    override val type = "com.eclipsesource.tabris.gcm.heartbeat.GCMHeartbeat"
+    override fun create(id: String, properties: V8Object) = GCMHeartbeat(scope)
+    override fun call(gcmHeartbeat: GCMHeartbeat, method: String, properties: V8Object) = when (method) {
+        "sendHeartbeat" -> gcmHeartbeat.sendHeartbeat()
+        else -> null
```
Comment (mpost) [inline src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt:13]: The else case is not required here.

File: src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.eclipsesource.tabris.android.ActivityScope
+import com.eclipsesource.tabris.android.ObjectHandler
+import com.eclipsesource.v8.V8Object
+
+class GCMHeartbeatHandler(private val scope: ActivityScope)
+    : ObjectHandler<GCMHeartbeat> {
+    override val type = "com.eclipsesource.tabris.gcm.heartbeat.GCMHeartbeat"
+    override fun create(id: String, properties: V8Object) = GCMHeartbeat(scope)
+    override fun call(gcmHeartbeat: GCMHeartbeat, method: String, properties: V8Object) = when (method) {
```
Comment (mpost) [inline src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt:11]: You should leave a blank line around the `type` field as well as between the methods,

File: src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt (hunk 1 above)
Comment (cpetrov) [reply src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt:13]: The compiler says:
```
GCMHeartbeatHandler.kt: (14, 91): 'when' expression must be exhaustive, add necessary 'else' branch
```

@mpost Do you know what the issue might be?

Comment (mpost) [reply src/android/com/eclipsesource/tabris/gcm/heartbeat/GCMHeartbeatHandler.kt:13]: Ah ok. Than the else is required. If you use an `Enum` with all values used in the when, than it is not required.

**Pull request merged**